README.md

```diff
@@ -217,23 +217,34 @@ kubectl apply -f https://raw.githubusercontent.com/<org>/cluster-api-agent/<tag
 
 E2E tests can be executed through ansible tasks in a target remote host.
 
-The following env vars are needed:
-`SSH_KEY_FILE` key file to access the remote host
-`REMOTE_HOST` remote host where to execute the tests
-`PULLSECRET` base64 encoded pullsecret to inject into the tests
+#### Prerequisites
 
-Then we can run:
+Export the following env vars:
+
+`SSH_KEY_FILE` path to the private SSH key file to access the remote host
+`SSH_AUTHORIZED_KEY` value of your public SSH key
+`REMOTE_HOST` remote host name where to execute the tests: `root@<REMOTE_HOST>`
+`PULLSECRET` base64-encoded pull secret to inject into the tests
+`DIST_DIR` is this repository directory `/dist` i.e. `$(pwd)/dist`
+`CONTAINER_TAG` set this to "local"
```

Comment:
Should we change the behaviour so that the user can decide what value this should have?

Reply:
Ideally, yes! I do think we should have this documented for now since it is currently what's needed to run the ansible playbook, right?

Reply:
Sure, I meant since it's a terrible way of doing it, we could've made it better while documenting :)
but I'm ok just documenting as is and improve it in the future

The issue is that [right now it can only be set to local](https://github.com/openshift-assisted/cluster-api-agent/blob/master/test/ansible/run_test.yaml#L99)

Reply:
Oh, I see what you mean. I thought you were asking to make everything configurable and didn't realize you were talking about just `CONTAINER_TAG`. Sorry! 

Sure we can definitely make `CONTAINER_TAG` a user-defined value! Adding to this PR in a moment